sys/net/gnrc/network_layer/ipv6/nib/nib_abr.c

```diff
@@ -34,7 +36,8 @@ int gnrc_ipv6_nib_abr_add(const ipv6_addr_t *addr)
     }
     abr->valid_until = 0U;
     while ((offl = _nib_offl_iter(offl))) {
```

Comment:
So this attaches all the off-link prefixes on the same interface to the new authoritative border router?
A comment would be good here.

Reply:
I just looked at the code again and I am not really sure what I was thinking... `addr` is the address of the border router so `gnrc_netif_get_by_ipv6_addr(addr)` will return NULL, as `addr` is not assigned to any interface. That might explain, why this did not work for you.